tests/generic_config_updater/test_lo_interface.py

```diff
@@ -0,0 +1,209 @@
+import logging
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.config_reload import config_reload
+from tests.generic_config_updater.gu_utils import apply_patch, expect_op_success, expect_op_failure
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+
+# Test on t0 topo to verify functionality and to choose predefined variable
+# "LOOPBACK_INTERFACE": {
+    # "Loopback0": {},
+    # "Loopback0|10.1.0.32/32": {},
+    # "Loopback0|FC00:1::32/128": {}
+# }
+pytestmark = [
+    pytest.mark.topology('t0'),
+]
+
+logger = logging.getLogger(__name__)
+
+@pytest.fixture(autouse=True)
+def setup_env(duthosts, rand_one_dut_hostname):
+    """
+    Setup/teardown fixture for each loopback interface test
+    Args:
+        duthosts: list of DUTs.
+        rand_selected_dut: The fixture returns a randomly selected DuT.
+    """
+    duthost = duthosts[rand_one_dut_hostname]
+
+    yield
+
+    logger.info("Restoring config_db.json")
+    config_reload(duthost)
```

Comment:
>config_reload [](http://example.com/codeflow?start=4&length=13)

Could you also test the rollback feature, instead of restoring the config directly?

You may check the design doc github.com/Azure/SONiC/blob/master/doc/config-generic-update-rollback/SONiC_Generic_Config_Update_and_Rollback_Design.md#223-rollback